src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String WHITE_GAP = " ";
+    private static final String COMMA = ",";
+    private static final String RESULT = "result";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final int PARSED_INDEX_0 = 0;
+    private static final int PARSED_INDEX_1 = 1;
+    private static final int COUNTER_INIT_VALUE = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] arrFromFile;
+        try {
+            arrFromFile = getArrFromFile(fromFileName);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file " + fromFileName);
+        }
+        String[] arrWithResult = getArrWithResult(arrFromFile);
+        try {
+            putResultToFile(toFileName, arrWithResult);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write into the file " + toFileName);
+        }
+    }
+
+    private void putResultToFile(String toFileName, String[] arrWithResult) throws IOException {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
```

Comment:
same here, you are writing try block, but don't catch anything, it's better to catch exception here